index.html

```diff
@@ -774,7 +789,7 @@ <h2>The <dfn><code>touch-action</code></dfn> CSS property</h2>
 
             <p>The <code>touch-action</code> CSS property determines whether <a>direct manipulation</a> interactions (which are not limited to touch, despite the property's name) MAY trigger the user agent's panning and zooming behavior. See the section on <a href="#details-of-touch-action-values"><code>touch-action</code> values</a>.</p>
 
-            <p>While panning and zooming, the user agent MUST NOT fire subsequent pointer events for the pointer. In order to end the stream of events for the pointer, the user agent MUST <a>fire a pointer event</a> named <code>pointercancel</code> (and subsequently a <code>pointerout</code> event and one or more <code>pointerleave</code> events) whenever all of the following are true:</p>
+            <p>Right before starting panning and zooming, the user agent MUST <a>suppress a pointer event stream</a> if all of the following conditions are true:</p>
```

Comment:
`Right before starting panning and zooming` perhaps better `Right before starting to pan and zoom`